lib/modules/manager/gitlabci-include/extract.ts

```diff
@@ -24,12 +25,9 @@ function extractDepFromIncludeFile(includeObj: {
   return dep;
 }
 
-export function extractPackageFile(
-  content: string,
-  _packageFile: string,
-  config: ExtractConfig
-): PackageFile | null {
+export function extractPackageFile(content: string): PackageFile | null {
```

Comment:
shouldn't this function match the manager interface? even there are unused parameters? 
just for API consistency.

Reply:
I'm not sure if I misunderstand, but don't we do the same as this for many/most managers already? e.g.

<img width="364" alt="image" src="https://user-images.githubusercontent.com/6311784/171786136-536861d1-6b26-45a1-96f1-c37cb7054177.png">

Reply:
ok, 🤔